org.kde.kwave.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "org.kde.kwave",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "kwave",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "libaudiofile",
+            "buildsystem": "autotools",
+            "config-opts": ["--disable-docs", "--disable-examples"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mpruett/audiofile",
+                    "commit": "b62c902dd258125cac86cd2df21fc898035a43d3"
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/pkgconfig",
+                "/share/man",
+                "*.la",
+                "*.a"
```

Comment:
these common ones should be in top level cleanup, so that you don't have to repeat.